posthog/clickhouse/migrations/0110_log_entries_v2_test.py

```diff
@@ -0,0 +1,12 @@
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.clickhouse.log_entries_v2_test import (
+    KAFKA_LOG_ENTRIES_V2_TABLE_SQL,
+    LOG_ENTRIES_V2_TABLE_SQL,
+    LOG_ENTRIES_V2_TABLE_MV_SQL,
+)
+
+operations = [
+    run_sql_with_exceptions(KAFKA_LOG_ENTRIES_V2_TABLE_SQL()),
+    run_sql_with_exceptions(LOG_ENTRIES_V2_TABLE_SQL()),
+    run_sql_with_exceptions(LOG_ENTRIES_V2_TABLE_MV_SQL()),
```

Comment:
Since this is not sharded, we need to add the `LOG_ENTRIES_V2_TABLE_SQL` to the coordinator. Kafka table and MV are not needed, since the insertion is made only in the data nodes.

Also, do not run them with `ON CLUSTER`, since the statement will be issued to all needed nodes.

```suggestion
    run_sql_with_exceptions(KAFKA_LOG_ENTRIES_V2_TABLE_SQL(on_cluster=False)),
    run_sql_with_exceptions(LOG_ENTRIES_V2_TABLE_SQL(on_cluster=False, node_role=NodeRole.ALL)),
    run_sql_with_exceptions(LOG_ENTRIES_V2_TABLE_MV_SQL(on_cluster=False)),
```